.travis.yml

```diff
@@ -10,6 +10,7 @@ cache:
 
 install:
     - pip install pipenv
+    - pip install --upgrade numpy 
```

Comment:
Why do you have to upgrade it here?  Isn't it being upgraded in the pipfile.lock enough?

Reply:
Unfortunately not: Travis ignores the NumPy version in the lockfile. Luckily this is only a temporary workaround until ras-secure-message migrates to GitHub Actions.

Reply:
Ah that'll be fine then.  We're replacing travis with github actions soon enough anyway so this workaround is fine.

Reply:
Actually ignore that.  Can you try and upgrade the OS image it uses?  That numpy addition to pipfile will probably get forgotten about.

Reply:
John and Warren removed Travis config just now, so the problem has evaporated. :-)